src/index.js

```diff
@@ -18,6 +18,7 @@ async function main() {
       setPagesPath({ staticSiteGenerator, generatorConfigFile, path: siteUrl.pathname })
     }
     outputPagesBaseUrl(siteUrl)
+    core.exportVariable('GITHUB_PAGES', true)
```

Comment:
We had the discussion with the team and we are fine having a global environment variable exposed here.